prac_05/wimbledon.py

```diff
@@ -0,0 +1,50 @@
+def read_wimbledon_data(filename):
+    """Read the Wimbledon champions data from a CSV file."""
+    champions_data = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        next(in_file)
+        for line in in_file:
+            data = line.strip().split(',')
+            year = data[0]
+            country = data[1]
+            champion = data[2]
+            champions_data.append((year, country, champion))
+    return champions_data
+
+
+def count_champions(champions_data):
+    """Count the number of titles for each champion."""
+    champion_count = {}
+    for _, _, champion in champions_data:
+        if champion in champion_count:
+            champion_count[champion] += 1
+        else:
+            champion_count[champion] = 1
+    return champion_count
+
+
+def get_countries(champions_data):
+    """Get a sorted set of countries of the champions."""
```

Comment:
Since the function returns a sorted list, you can name the function "get_sorted_countries".

Reply:
Thank you for your feedback, it has helped me a lot with my programming skills